README.md

```diff
@@ -144,6 +144,10 @@ The following configuration options can be set as environment variables or in a
 - `WORK_DIR`: directory to store NWC data files. Default: $XDG_DATA_HOME/albyhub
 - `LOG_LEVEL`: log level for the application. Higher is more verbose. Default: 4 (info)
 
+## Node-specific backend parameters
+
+- `ENABLE_ADVANCED_SETUP`: set to `false` to force a specific backend type (combined with backend parameters below)
```

Comment:
If a specific backend is configured, do we still need to provide other setup options? In what environment would that be the case? :thinking:

Reply:
In Umbrel for example we allow the user to go to advanced and use an LDK node instead of the default (LND)

On Nodana however, they want to force a node type (Phoenixd)